src/test/java/bio/terra/app/controller/DatasetsApiControllerTest.java

```diff
@@ -358,16 +362,16 @@ void testCreateCriteriaData(
   @Test
   void testCreateSnapshotRequest() throws Exception {
     mockValidators();
-    SnapshotAccessRequest expected = SnapshotBuilderTestData.createSnapshotAccessRequest();
+    SnapshotAccessRequest request = SnapshotBuilderTestData.createSnapshotAccessRequest();
```

Comment:
Was there a request to change this name? I think `expected` is clearer here than `request` as `expected` refers to what the object's use is, rather than its type.

Reply:
I changed the name to be more clear. This is not the expected return type. This is the request given to get the expected response. I think it would be even clearer if I updated the name of `response` to `expected`